src/vpc.cpp

```diff
@@ -408,6 +410,7 @@ void buildVpc(std::vector<std::string> args)
     programArgs.add("boundary", "Calculate boundary polygons from data", boundaries);
     programArgs.add("stats", "Calculate statistics from data", stats);
     programArgs.add("overview", "Create overview point cloud from source data", overview);
+    programArgs.add("input_file_list", "Read input files from a txt file, one file per line.", inputFileList);
```

Comment:
Can we change the argument name, so that it is consistent with other PDAL wrench algs?
```suggestion
    programArgs.add("input-file-list", "Read input files from a txt file, one file per line.", inputFileList);
```

Reply:
sure! updated in last commit.